app/services/queries/reviews_turnaround_per_user_project_query.rb

```diff
@@ -7,26 +7,15 @@ def initialize(time_interval:)
     end
 
     def call
-      run
+      execute
     end
 
     private
 
-    def query
+    def execute
       Events::Review.joins(:review_request, owner: :users_projects, pull_request: :project)
                     .includes(:review_request)
                     .where(opened_at: @time_interval.starting_at..@time_interval.ending_at)
```

Comment:
the weird part is that this query object know how is it going to be used, I mean because of the `includes` and `joins`

Reply:
Yep, you are right. I will make that change on other PR